### ISSUE TITLE
Empty UI after compiling in Visual Studio 2015

### ISSUE THREAD

**Issue by maxruby** [braitsch/ofxDatGui#78]
This is a great addon! However, I have been trying to get several examples from ofxDatGui to run on Visual Studio 2015 (Windows 10), and in all cases after successfully compiling I see only an empty Window or UI missing the text and icons. Everything is set up as per the instructions given here. I suspect at least in part the font and icons may not be loaded properly because of the paths.  I tried adding a custom Asset path in ofApp.h as follows:
 
```c++
std::string path; 
path = "C:/Users/mxsst/openFrameworks/of_v0.9.3_vs_release/of_v0.9.3_vs_release/apps/myApps/ofDatGuiHacking/bin/data/ofxbraitsch/";
ofxDatGui::setAssetPath(path);
```
I also looked into `ofxDatGuiTheme.h`, and I can see that AssetPath is a static string that is used to set the full path of the font and/or icons. Also tried changing it here but it did not fix the empty UI issue.

Any tips on how to get this working on Visual Studio 2015?

**Comment by braitsch** [comment]
Hi, there's an issue related to ``ofxDatGuiTheme::AssetPath`` that appears to be causing fonts & images to not load correctly. Try passing an empty string trying into ``ofxDatGui::setAssetPath("")`` and let me know if that works for you.

**Comment by maxruby** [comment]
Thanks for the quick feedback.  I added `ofxDatGui::setAssetPath("")` in `ofApp::setup()` before creating the GUI element.  However, the UI is still empty and I see the following output in the console (as before):

```shell
ofxSmartFont :: new font added : Verdana.ttf @ pt size 6
```

**Comment by braitsch** [comment]
If you're seeing that ofxSmartFont notice in the console that means the font file was found and loaded correctly. Can you run the AllComponentsGui example and provide a screenshot?

**Comment by maxruby** [comment]
Thanks again.  I found out that the issue was actually due to starting the app in fullscreen mode:
```c++
	// launch the app //
	mFullscreen = true;
```
After pressing `f` to toggle the screen mode selection, I managed to see the control panel and the screen. However, toggling again (i.e., pressing 'f') to the fullscreen mode results in flashing of the screen and the control panel dissapears. 

For now, I have to set the following for it to work: 
```c++ 
mFullscreen = false;
```

It seems this might originate from incorrect detection and/or handling of screen dimensions in `ofAppRunner.cpp` after you call `ofGetScreenWidth()` and `ofGetScreenHeight()` in `ofxDatGuiTheme.h` ?
